### PR TITLE
Tickets/dm 22531

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: AT_CSC/atdome_setup.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+setup lsst_distrib
+echo "#"
+echo "# Loading sal environment"
+. /home/saluser/repos/ts_sal/setup.env
+echo "#"
+echo "# Setting ATDome Simulator"
+
+cd ~/repos/ts_config_attcs/
+git fetch --all
+git checkout tickets/DM-22531
```
Comment (patrickingraham) [inline AT_CSC/atdome_setup.sh]: has this been merged or do we still want to use this branch? Once you merge this and delete this branch won't it break the file?

**Review by patrickingraham — COMMENTED** [review]
Permission to merge based upon the answer to question...

File: AT_CSC/atdome_setup.sh (hunk 1 above)
Comment (tribeiro) [reply AT_CSC/atdome_setup.sh]: Good catch! let me get the config merged so I can fix this!

Comment (tribeiro) [reply AT_CSC/atdome_setup.sh]: Done! I'm using the new release tag. I'll create a ticket to re-do the deployment of the summit components next week with clean environment.

**Review by patrickingraham — APPROVED** [review]

**Review by patrickingraham — APPROVED** [review]

**Review by couger01 — APPROVED** [review]
LGTM

**Pull request merged**